dist/s_export.list

```diff
@@ -21,3 +21,5 @@ wiredtiger_unpack_start
 wiredtiger_unpack_str
 wiredtiger_unpack_uint
 wiredtiger_version
+# MSan instrumentation that appears when -fsanitize-memory-track-origins is used.
+__msan_track_origins
```

Comment:
Does the s_all execute on MSAN builds?

Reply:
`s_all` is part of the PR testing, without this addition, you would get:

```
####################### MESSAGE ############################
s_all run of: "./s_export" resulted in:
    =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
    unexpected external symbols in the WiredTiger library ../build_msan/libwiredtiger.so
    =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
    __msan_track_origins
#######################
```

Reply:
Thanks!